frontend/ManagementController.cs

```diff
@@ -39,7 +39,7 @@ public async Task ScaleOut()
         {
             _log.LogInformation("Scaling out the actor cluster");
 
-            var azure = await GetAzureContext();
+            var azure = await GetAzureContext().ConfigureAwait(false);

```

Comment:
i always felt like this `ConfigureAwait(false)` should have been implemented as a different `await` keyword like `await-any`. I felt frustrations of having to scatter this all over codebases in the past coming back. Not to mention in most cases this behavior is perfectly acceptable over the opposite.

Reply:
yep, always takes away (some of) the magic of await, for me. sigh.